.idea/misc.xml

```diff
@@ -1,4 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
do we need this file at all?

Reply:
@lfmoisa-globality can you remove the folder? then I'll check with Salman and we can probably merge